services/data/src/links/RestAPILink/queryToRequestOptions/textPlainMatchers.test.ts

```diff
@@ -273,3 +274,20 @@ describe('isExpressionDescriptionValidation', () => {
         ).toBe(false)
     })
 })
+
+describe('isFilterDescriptionValidation', () => {
+    it('returns true for a POST to "programIndicators/filter/description"', () => {
+        expect(
+            isFilterDescriptionValidation('create', {
+                resource: 'programIndicators/filter/description',
+            })
+        ).toBe(true)
+    })
+    it('retuns false for a POST to a different resource', () => {
+        expect(
+            isMetadataPackageInstallation('create', {
```

Comment:
I think there is a small copy-paste issue here...
```suggestion
            isFilterDescriptionValidation('create', {
```

Reply:
Fixed with the other occurrences in other tests.